tailwind.config.js

```diff
@@ -32,6 +32,26 @@ module.exports = {
         "1/20": "5%",
         "1/100": "1%",
       },
+      spacing: {
+        "10/100": "10%",
+        "15/100": "15%",
+        "20/100": "20%",
+        "25/100": "25%",
+        "30/100": "30%",
+        "35/100": "35%",
+        "40/100": "40%",
+        "45/100": "45%",
+        "50/100": "50%",
+        "55/100": "55%",
+        "60/100": "60%",
+        "65/100": "65%",
+        "70/100": "70%",
+        "75/100": "75%",
+        "80/100": "80%",
+        "85/100": "85%",
+        "90/100": "90%",
+        "95/100": "95%",
```

Comment:
We are using a few.
I'll remove the ones not used.
And as a personal preference, I would like using 60/100 instead of 3/5 (an existing class)
Because everytime I want to add something, I'll have to go check and see what tailwind already provides.
Too much hassle. 60/100 is super straight and simple